cluster/Dockerfile

```diff
@@ -2,7 +2,36 @@ FROM ubuntu:16.04
 
 ARG DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update && apt-get install -f -y pkg-config python3 python3-pip libffi6 libfreetype6 libopenblas-base libffi-dev libfreetype6-dev libpng-dev libopenblas-dev curl
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        build-essential \
+        curl \
+        git \
+        libcurl3-dev \
+        libfreetype6-dev \
+        libpng12-dev \
+        libzmq3-dev \
+        pkg-config \
+        rsync \
+        software-properties-common \
+        unzip \
+        zip \
+        zlib1g-dev \
+        fuse \
+        python3 \
+        python3-numpy \
+        python3-dev \
+        python3-pip \
+        python3-wheel \
+        python3-setuptools \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN curl -v -L -O https://github.com/GoogleCloudPlatform/gcsfuse/releases/download/v0.23.0/gcsfuse_0.23.0_amd64.deb && \
```

Comment:
do we still use GCSfuse?  I think this can go away, as can the uuid-runtime

Reply:
Ah, cool. I was making the CPU dockerfile consistent with the GPU settings. I'll remove the fuse stuff from both. And the uuid-runtime

Reply:
Oh!  does the CPU one still build?  The CPU one is a bit more complicated because it has to build tensorflow from source in the container to have the CPU optimizations, whereas the GPU one gets all the magic from `pip install tensorflow-gpu`